t/op/svleak.t

```diff
@@ -511,7 +510,6 @@ leak(5, 0, sub { scalar &const_av_xsub_leaked }, "const_av_sub in scalar context
 # check that OP_MULTIDEREF doesn't leak when compiled and then freed
 
 eleak(2, 0, <<'EOF', 'OP_MULTIDEREF');
-no strict;
```

Comment:
we should probably leave that `no strict`
it's already like this in blead, and we should not remove it IMO

Reply:
> we should probably leave that `no strict`
> it's already like this in blead, and we should not remove it IMO

What if it was never really needed in blead?

Reply:
then my advice would be let's fix it in blead and not there